test/reporters/base.spec.js

```diff
@@ -136,6 +136,36 @@ describe('Base reporter', function () {
     });
   });
 
+  describe('Diff generation', function () {
+    it('should generate unified diffs if `inlineDiff === false`', function () {
+      var actual = 'a foo unified diff';
+      var expected = 'a bar unified diff';
+
+      var oldInlineDiffs = Base.inlineDiffs;
+      Base.inlineDiffs = false;
+
+      var output = Base.generateDiff(actual, expected);
```

Comment:
please do the patching in a `beforeEach` of the suite and restore in `afterEach`

Reply:
When doing the patching, should I be setting the value of `Base.inlineDiffs` in the `beforeEach`?